lib/lightning/projects.ex

```diff
@@ -236,4 +236,150 @@ defmodule Lightning.Projects do
 
     {:ok, yaml}
   end
+
+  @spec import_project(any, any) :: {:ok, binary}
+  @doc """
+  Imports a project as map.
+
+  ## Examples
+
+      iex> import_project(:yaml, project_id)
+      {:ok, string}
+
+  """
+  def import_project(project_data, user) do
+    ImportUtils.import_multi_for_project(project_data, user)
+    |> Repo.transaction()
+  end
+end
+
+defmodule ImportUtils do
```

Comment:
@wanecode I think this module should be under `Projects`, right now it's at the root namespace.

So it should be either `Lightning.Projects.ImportUtils`, but I think it might be better named as `Lightning.Projects.Importer`.

And with that, can it be moved to it's own file under `lightning/projects/importer.ex`?